package.json

```diff
@@ -50,7 +50,19 @@
 					"scope": "resource",
 					"type": "string",
 					"default": "qcc",
-					"description": "Controls parser support for various language features."
+					"description": "Automatically assigns defaults for compiler feature flags based on a specific language."
+				},
+				"quakec.features": {
+					"scope": "resource",
+					"type": "object",
+					"description": "Feature flags for specific parser features.",
```

Comment:
We should probably change our verbiage to "Language features". The term "Feature Flags" means something different.